velox/core/PlanNode.cpp

```diff
@@ -152,6 +190,96 @@ void AggregationNode::addDetails(std::stringstream& stream) const {
   }
 }
 
+// static
+const char* AggregationNode::stepName(AggregationNode::Step step) {
+  switch (step) {
+    case Step::kPartial:
+      return "PARTIAL";
+    case Step::kFinal:
+      return "FINAL";
+    case Step::kIntermediate:
+      return "INTERMEDIATE";
+    case Step::kSingle:
+      return "SINGLE";
+    default:
```

Comment:
nit: nice thing about not having a default branch is that the compiler would fail if someones adds a new value to the enum but not a branch to this switch. 

Maybe remove it and instead just a have a `return "";` which would never get reached but is needed to make gcc happy?